docs/documentation/actions/kafka.md

```diff
@@ -0,0 +1,189 @@
+    |:--------:|:------------------|:---------------------------------|:-------------------------------|:-------------------------------------------------------|
+    |    *     | `target`          | {name: string, url: string}      |                                | Kafka target                                           |
+    |    *     | `topic`           | String                           |                                | Topic from where the message will be consumed          |
+    |    *     | `group`           | String                           |                                | Group id of the consumer                               |
+    |          | `properties`      | Map <String, String>             |                                | [Configurations](https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java#:~:text=CONFIG%20%3D%20new,TRANSACTIONAL_ID_DOC){:target="_blank"} for Kafka producer. |
+    |          | `header-selector` | String                           |                                | Header Selector                                         |
+    |          | `selector`        | String                           |                                | Message Selector [json paths](https://github.com/json-path/JsonPath){:target="_blank"} or [xml paths](https://www.w3schools.com/xml/xml_xpath.asp){:target="_blank"} |
+    |          | `nb-messages`     | Integer                          | 1                              | How many messages to be consumed                       |
+    |          | `content-type`    | String                           | `application/json`             | To be consumed message's content type                  |
+    |          | `timeout`         | [duration](/documentation/actions/other/#duration-type) | `60 sec`                       | In how many time Kafka target must return the message  |
+    |          | `ackMode`         | [AckMode](https://docs.spring.io/spring-kafka/api/org/springframework/kafka/listener/ContainerProperties.AckMode.html) {:target="_blank"} | target's ackMode, else `BATCH` | The offset commit behavior   |
+
+=== "Outputs"
+
+    |    Name    | Type     | Description                           |
+    |:-----------|:---------|:--------------------------------------|
+    | `body`     | String   | Consumed messages                     |
+    | `payloads` | List     | Consumed messages payloads            |
+    | `headers`  | List     | Consumed messages headers             |
+
+### Example
+
+=== "Kotlin"
+``` kotlin
+KafkaBasicConsumeAction(
+    target = "my_kafka_target",
+    topic = "my.queue",
+    group= "my.group",
+    timeout= "10 sec",
+    selector= "\$..[?(\$.payload.title==\"Castle in the Sky\")]",
+    headerSelector= "\$..[?(\$.headers.season=='1')]",
+    contentType= "application/json"
+)
+```
+
+# Embedded Kafka server
```

Comment:
I thought we won't document this quite heavy and troublesome action ?
It just mandatory for our integration tests and should be delete with the use of Testcontainers.